### PR TITLE
Fixes for cache_and_resume.md

### PR DESCRIPTION
**Pull request by pinin4fjords**
A couple of fixes:

- Dollar signs are unhelpful in command blocks (they have to be removed by anyone copying/ pasting the blocks)
- Final example is hard to talk through, because it's incomplete- suggesting a fix

In that final example the 'solution' part does not match the 'problem' part, which makes it hard to follow. There's also a lot of 'imagining' required. I've made a simpler, more complete example for both.

### CODE REVIEW

**Review by kenibrewer — APPROVED** [review]
Good improvement!

**Review by maxulysse — APPROVED** [review]

**Review by kenibrewer — APPROVED** [review]
